lib/mayaUsd/ufe/private/Utils.cpp

```diff
@@ -21,13 +21,37 @@
 #include <pxr/usd/usd/primCompositionQuery.h>
 #include <pxr/usd/usdGeom/xformable.h>
 
+#include <maya/MGlobal.h>
 #include <ufe/log.h>
 
 #include <memory>
 #include <string>
 
 PXR_NAMESPACE_USING_DIRECTIVE
 
+namespace {
+// find position index for a given layer in the local LayerStack ( strong-to-weak order )
+uint32_t findLayerIndex(const PXR_NS::UsdAttribute& attr, const PXR_NS::SdfLayerHandle& layer)
+{
+    const auto& prim = attr.GetPrim();
+    const auto& stage = prim.GetStage();
+    const auto& layerStack = stage->GetLayerStack();
+
+    uint32_t position { 0 };
+
+    auto iter = std::find_if(
```

Comment:
I believe std::find_if is what I wanted to use there since "std::find_if" takes a unary predicate and checks whether the predicate returns true or false to a given element. However you are right, the conditional find could be replace by this:

auto iter = std::find(std::begin(layerStack), std::end(layerStack), layer);

Both ways would work. Addressed in fe8131c1c837ebf7bf5405e7c594df7b5b7b1db4